### PR TITLE
Fix some misspelled words

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: docs/CONFIG.md [hunk 1]
```diff
@@ -212,8 +212,8 @@ of tokens before they browse.
 
 ### config\["get-more-passes-url"\]
 
-A valid URL string specifying an url where users can get more signed tokens by solving a 
-challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. 
+A valid URL string specifying an url where users can get more signed tokens by solving a
```
Comment (Lekensteyn) [inline docs/CONFIG.md]: While at it, maybe `an url` -> `a URL`?

Comment (claucece) [reply docs/CONFIG.md]: For sure! Just did it and squash the commits ;)

**Comment by alirezajavani** [comment]
@ of tokens before they browse.
### config\["get-more-passes-url"\]
A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

 Lekensteyn • about 2 months ago

While at it, maybe an url -> a URL?

 

 claucece • about 2 months ago

For sure! Just did it and squash the commits ;)

 

Reply…

claucece added a commit about 2 months ago

Fix some misspelled words and uppercase

claucece force pushed changes to this branch about 2 months ago

**Comment by alirezajavani** [comment]
privacypass / challenge-bypass-extension

Code Issues 13 Pull requests 1 Projects 0 Actions Wiki Security Pulse Community

ConversationCommitsChanges

Jump to bottom Open

Fix some misspelled words #148

claucece:master

claucece opened this pull request 
about 2 months ago

No description given.

 

Lekensteyn reviewed about 2 months ago

Show outdateddocs/CONFIG.md

claucece added a commit about 2 months ago

Fix some misspelled words and uppercase

claucece force pushed changes to this branch about 2 months ago

alirezajavani 
commented less than a minute ago

@ of tokens before they browse.

config["get-more-passes-url"]

A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up Get more passes menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

 Lekensteyn • about 2 months ago

While at it, maybe an url -> a URL?

 

 claucece • about 2 months ago

For sure! Just did it and squash the commits ;)

 

Reply…

claucece added a commit about 2 months ago

Fix some misspelled words and uppercase

claucece force pushed changes to this branch about 2 months ago

 

Comment on pull request

Comment

Notifications for this thread

You’re receiving notifications because you commented.

Notifications

Customize

 Unsubscribe

Desktop versionSign out

**Comment by alirezajavani** [comment]
privacypass / challenge-bypass-extension

Code Issues 13 Pull requests 1 Projects 0 Actions Wiki Security Pulse Community

ConversationCommitsChanges

Jump to bottom Open

Fix some misspelled words #148

claucece:master

claucece opened this pull request 
about 2 months ago

No description given.

 

Lekensteyn reviewed about 2 months ago

Show outdateddocs/CONFIG.md

claucece added a commit about 2 months ago

Fix some misspelled words and uppercase

claucece force pushed changes to this branch about 2 months ago

alirezajavani 
commented less than a minute ago

@ of tokens before they browse.

config["get-more-passes-url"]

A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up Get more passes menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

 Lekensteyn • about 2 months ago

While at it, maybe an url -> a URL?

 

 claucece • about 2 months ago

For sure! Just did it and squash the commits ;)

 

Reply…

claucece added a commit about 2 months ago

Fix some misspelled words and uppercase

claucece force pushed changes to this branch about 2 months ago

 

Comment on pull request

Comment

Notifications for this thread

You’re receiving notifications because you commented.

Notifications

Customize

 Unsubscribe

Desktop versionSign out

**Comment by alirezajavani** [comment]
@@ -212,8 +212,8 @@ of tokens before they browse.
### config\["get-more-passes-url"\]
A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

**Comment by alirezajavani** [comment]
https://github.com/privacypass/challenge-bypass-extension/pull/148#issuecomment-582555572

**Comment by alirezajavani** [comment]
https://github.com/privacypass/challenge-bypass-extension/pull/148#issuecomment-582556233

**Comment by alirezajavani** [comment]
FIG.md

@@ -212,8 +212,8 @@ of tokens before they browse.### config\["get-more-passes-url"\]A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

**Comment by alirezajavani** [comment]
@@ -212,8 +212,8 @@ of tokens before they browse.### config\["get-more-passes-url"\]A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

**Comment by alirezajavani** [comment]
https://github.com/privacypass/challenge-bypass-extension/pull/148#discussion_r360722340

**Comment by alirezajavani** [comment]
before they browse.### config\["get-more-passes-url"\]A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

 Lekensteyn • a

**Comment by alirezajavani** [comment]
before they browse.### config\["get-more-passes-url"\]A valid URL string specifying an url where users can get more signed tokens by solving a challange/CAPTCHA. This URL is pulled into the pop-up `Get more passes` menu item. A valid URL string specifying an url where users can get more signed tokens by solving a

 Lekensteyn • a

**Pull request merged**